modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -15711,6 +15712,8 @@ sites-tab-description=Sites can only be assigned to a single property at a time.
 sites-that-i-administer=Sites That I Administer
 sites-to-sync-x=Sites to Sync - {0}
 size=Size
+size-limit-copy-files-title=Size limit for copying files
+size-limit-copy-files-help=Set the maximum size for copying files in bytes. The total number of copied files must not exceed the value specified in this property to complete the copying.
```

Comment:
I'd change this to: "Set the maximum file size in bytes for copying files. The total size of all copied files must not exceed this value."

Reply:
Changed! Thanks